config/configuration.json

```
@@ -315,6 +315,24 @@
       "tags":"coastline"
     }
   },
+  "GenericTagCheck": {
+    "db": {
+      "taginfo": "extra/taginfo-db.db",
+      "wikidata": "extra/wikidata.db"
+    },
+    "tag": {
+      "usage.min": 100,
+      "percentage_of_key_for_popular": 10
+    },
+    "wikidata.tag_removal": ["abandoned", "deprecated", "imported", "obsolete", "rejected"],
+    "challenge": {
+      "description": "Tasks containing features with tags containing missing, conflicting, incorrect or illegal values",
+      "blurb": "Features with invalid tags",
+      "instruction": "Open your favorite editor and check that the listed tags are correct.",
+      "difficulty": "Medium",
```

Can you change this to "EXPERT"?
```suggestion
      "difficulty": "EXPERT",
```
I think that we want this to be noted as an expert challenge and if I am reading the code correctly the correct value is "EXPERT".
https://github.com/osmlab/atlas-checks/blob/dev/src/main/java/org/openstreetmap/atlas/checks/maproulette/data/ChallengeDifficulty.java#L10

---

And done. :)